apps/src/code-studio/components/progress/TeacherPanel.jsx

```diff
@@ -57,6 +58,9 @@ class TeacherPanel extends React.Component {
     getSelectedUserId: PropTypes.func,
     sectionData: PropTypes.object,
     scriptName: PropTypes.string,
+    // pageType describes the current route the user is on (e.g., 'level',
+    // 'script_overview', 'stage_extras'). Used only for logging to Firehose.
+    pageType: PropTypes.string,
```

Comment:
since this is a limited set should we do something like `PropTypes.oneOf(['level', 'script_overview', 'stage_extras']),